hexrdgui/masking/mask_manager.py

```diff
@@ -30,11 +30,13 @@ def __init__(
         visible=True,
         show_border=False,
         mode=None,
-        xray_source=None
+        xray_source=None,
+        highlight=False
```

Comment:
```suggestion
        highlight=False,
```

I'd recommend including a `,` at the end of all final arguments. The reason is that, without it, a new change is necessary in the changelog to add a comma to the previous one (as is the case here for `xray_source=None`). This can cause confusion as to who authored that line and why it was authored.